src/store/diagnostics.ts

```diff
@@ -3,6 +3,7 @@ import { GetterTree } from 'vuex';
 
 import { ActionContext, MutationsType } from './ts-helpers';
 
+import { Settings } from '@/config/settings';
```

Comment:
```suggestion
import type { Settings } from '@/config/settings';
```
Don't want to end up pulling in weird things into the renderer process…